src/main/java/de/adesso/kicker/notification/MatchVerificationRequest/VerifyMatchService.java

```diff
@@ -0,0 +1,78 @@
+package de.adesso.kicker.notification.MatchVerificationRequest;
+
+import de.adesso.kicker.match.Match;
+import de.adesso.kicker.user.User;
+import de.adesso.kicker.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class VerifyMatchService {
+
+    private MatchVerificationRequestRepository matchVerificationRequestRepository;
+
+    private UserService userService;
+
+    @Autowired
+    public VerifyMatchService(MatchVerificationRequestRepository matchVerificationRequestRepository,
+            UserService userService) {
+        this.matchVerificationRequestRepository = matchVerificationRequestRepository;
+        this.userService = userService;
+    }
+
+    public void acceptRequest(MatchVerificationRequest matchVerificationRequest) {
+        // TODO trigger event to verify match
```

Comment:
Comments shouldn't be used to mark when you still need to do something. You should use the issue tracker for such things.

Reply:
I can delet those coments, i just thought it wouldbe nice to mark where those future calls have be made so you dont have to read the whole code again